src/axom/core/Types.hpp

```diff
@@ -34,20 +50,21 @@ using uint16 = std::uint16_t; /*!< 16-bit unsigned integer type   */
 using int32 = std::int32_t;   /*!< 32-bit signed integer type     */
 using uint32 = std::uint32_t; /*!< 32-bit unsigned integer type   */
 
-// Note: KW -- We assume that AXOM_NO_INT64_T will be defined
-// on systems/compilers that do not support 64 bit integer types
-#ifndef AXOM_NO_INT64_T
+  // Note: KW -- We assume that AXOM_NO_INT64_T will be defined
+  // on systems/compilers that do not support 64 bit integer types
+  #ifndef AXOM_NO_INT64_T
 using int64 = std::int64_t;   /*!< 64-bit signed integer type     */
 using uint64 = std::uint64_t; /*!< 64-bit unsigned integer type   */
+  #endif
 #endif
 
 using float32 = float;
 using float64 = double;
```

Comment:
If we're defining and using `float32` and `float64`, presumably, we should also check that these are actually 32-bits and 64-bits, respectively.

Reply:
Created issue https://github.com/LLNL/axom/issues/1074